src/code/z_vimode.c

```diff
@@ -1,17 +1,331 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_vimode/func_800AC030.s")
+#define LOG_MODE_FIELD(exp, line) LOG_ADDRESS(#exp, exp, "../z_vimode.c", line)
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_vimode/func_800AC2F4.s")
+void ViMode_LogPrint(OSViMode* osvimodep) {
+    LOG_MODE_FIELD(osvimodep, 87);
+    LOG_MODE_FIELD(osvimodep->comRegs.ctrl, 88);
+    LOG_MODE_FIELD(osvimodep->comRegs.width, 89);
+    LOG_MODE_FIELD(osvimodep->comRegs.burst, 90);
```

Comment:
not sure about this custom macro that includes the vimode file string. We dont really do this anywhere else currently, feel like we can just use the LOG_ADDRESS macro as is.
can get some other opinions tho

Reply:
I switched to a plain `LOG_ADDRESS` macro.